modules/hackathon-2023/Hackathon2023TopBanner.tsx

```diff
@@ -0,0 +1,86 @@
+import { Grid, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { ReactElement } from 'react';
+
+import { darken } from '../../styles/helpers/color';
+
+const useStyles = makeStyles(
+  (theme) => ({
+    root: {
```

Comment:
I didn't do it in my PR since I think it's better to get the page up before making abstractions, but how would you feel about a "section" component that we can use instead of defining `root` classes?

Reply:
I have consistently used `root` for the parent component, should probably stick with it just for consistency? Is there any reason we wouldn't want to use `root` for this use case?